generators/fast-api/templates/docker-compose.yaml.ejs

```diff
@@ -0,0 +1,8 @@
+services:
+    <%= packageName %>:
+        build:
+            context: <%= packagePath %>/
+            volumes:
+            - ./src:/opt/pysetup
+        environment:
+            DATABASE_URL: "postgresql+psycopg2://user:password@postgres:5432"
```

Comment:
This should be contains the database name (which is the same as the package name by default).
```suggestion
            DATABASE_URL: "postgresql+psycopg2://user:password@postgres:5432/<%= packageName %>"
```